lib/land/trackers/api_tracker.rb

```diff
@@ -124,6 +125,13 @@ def maybe_update_visit_attribution
         visit.update(attribution:) unless attribution_values_present?(visit)
       end
 
+      def maybe_update_visit_attribution
```

Comment:
This method is defined twice on lines 120 and 128. Should this be `maybe_update_visit_referer`? This method is defined twice

Reply:
doh 🤭 - Thank you!